### ISSUE TITLE
Problem installing in Laravel 6.0

### ISSUE THREAD

**Issue by g4b0** [Laravel-Backpack/Base#407]
# Bug report
Installing backpack throug composer works fine. Running php artisan backpack:base:install fail with the following error:

### What I did:

```
# php artisan backpack:base:install
 0/8 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   0% Backpack\Base installation started. Please wait...
 1/8 [▓▓▓░░░░░░░░░░░░░░░░░░░░░░░░░]  12% Installing backpack/generators

   Symfony\Component\Process\Exception\ProcessFailedException  : The command "composer require backpack/generators --dev" failed.

Exit Code: 2(Misuse of shell builtins)

Working directory: /mnt/data/develop/

Output:
================


Error Output:
================
Using version ^2.0 for backpack/generators
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for backpack/generators ^2.0 -> satisfiable by backpack/generators[2.0.0, 2.0.1].
    - Can only install one of: backpack/crud[v4.x-dev, 3.6.31].
    - Can only install one of: backpack/crud[v4.x-dev, 3.6.31].
    - Can only install one of: backpack/crud[v4.x-dev, 3.6.31].
    - Conclusion: install backpack/crud v4.x-dev
    - Installation request for backpack/crud (locked at 3.6.31, required as ^3.6) -> satisfiable by backpack/crud[3.6.31].


Installation failed, reverting ./composer.json to its original content.


  at /mnt/data/develop/vendor/backpack/base/src/app/Console/Commands/Install.php:100
     96|         });
     97|
     98|         // executes after the command finishes
     99|         if (!$process->isSuccessful()) {
  > 100|             throw new ProcessFailedException($process);
    101|         }
    102|
    103|         if ($this->progressBar) {
    104|             $this->progressBar->advance();

  Exception trace:

  1   Backpack\Base\app\Console\Commands\Install::executeProcess("composer require backpack/generators --dev")
      /mnt/data/develop/vendor/backpack/base/src/app/Console/Commands/Install.php:52

  2   Backpack\Base\app\Console\Commands\Install::handle()
      /mnt/data/develop/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:32

  Please use the argument -v to see more details.
```

### Backpack, Laravel, PHP, DB version:

Laravel 6.0
PHP 7.3.9
Mysql 5.5

$ more composer.json | grep backpack
        "backpack/crud": "^3.6",

**Comment by g4b0** [comment]
Ok, solved installing Installing backpack/crud v4.x-dev

**Comment by tabacitu** [comment]
Thanks for raising the issue @g4b0 . I also pushed a hot fix for this in v3.6. So that one's fixed too.

Cheers!
